src/test/java/com/hillel/items_exchange/service/LocationServiceTest.java

```diff
@@ -145,4 +153,30 @@ void update_shouldUpdateLocation() {
                 () -> assertEquals(locationCaptor.getValue().getCity(), updatedLocationDto.getCity()),
                 () -> assertEquals(locationCaptor.getValue().getDistrict(), updatedLocationDto.getDistrict()));
     }
-}
\ No newline at end of file
+
+    @Test
+    void createFileToInitLocations_shouldReturnProperData()
+            throws IOException, InvalidLocationInitFileCreatingDataException {
+        String locationsString = Files.readString(pathToFileParseLocationsFrom);
+        String createdFileContent = locationService.createParsedLocationsFile(locationsString);
+        assertTrue(createdFileContent.contains("2580"));
```

Comment:
It's better to add **ArgumentCaptor<List<Location>>** that will capture them from **private List<Location> mapCreatingDataToLocations(String creatingData)** method or size from **locationStings** variable.